templates/patterns/button/button.ui_patterns.yml

```diff
@@ -1,89 +1,65 @@
 button:
-  label: Button
-  description: For actions in forms, dialogs, and more with support for multiple sizes, states, and more. https://v5.getbootstrap.com/docs/5.0/components/buttons/
-  variants:
-    primary:
-      label: Default
-    secondary:
-      label: Secondary
-      description: Demo secondary button.
-    warning:
-      label: Warning
-      description: Demo warning button.
+  label: 'Button'
+  description: 'For actions in forms, dialogs, and more with support for multiple sizes, states, and more. For more information: https://v5.getbootstrap.com/docs/5.0/components/buttons/'
   settings:
-    variant:
+    style:
       type: select
-      label: Button style
+      label: 'Button style'
+      description: 'Sets the background color and text color of the button:'
       options:
-        primary: primary
-        secondary: secondary
-        success: success
-        danger: danger
-        warning: warning
-        info: info
-        light: light
-        dark: dark
-        link: link
-      preview: primary
+        primary: 'primary (default)'
+        secondary: 'secondary'
+        success: 'success'
+        danger: 'danger'
+        warning: 'warning'
+        info: 'info'
+        light: 'light'
+        dark: 'dark'
+        link: 'link'
+      preview: 'primary'
     outline:
       type: boolean
-      label: Button outline
+      label: 'Button outline'
+      description: 'Lighter button style removing background color.'
       preview: false
```

Comment:
Not necessary, delete it please and if present, delete setting default value at markup rendering test.